src/findIndex.ts

```diff
@@ -85,10 +85,10 @@ export namespace findIndex {
   export function indexed<T>(
     array: readonly T[],
     fn: PredIndexed<T, boolean>
-  ): T | undefined;
+  ): number;
   export function indexed<T>(
     fn: PredIndexed<T, boolean>
-  ): (array: readonly T[]) => T | undefined;
+  ): (array: readonly T[]) => number;
```

Comment:
I found this while looking through the `findIndex` implementation, and it seemed like a bug. 🙂

Reply:
yes, thank you. probably a copy-paste error (looks like the `find` interface). `findIndex` returns a `number` 😅